src/search_datastructures.hpp

```diff
@@ -437,8 +427,11 @@ class GlobalDataHolder
     TBlastRepFile       outfileBlastRep;
     TBamFile            outfileBam;
 
-    TScoreScheme3       scoringSchemePreScoring;
-    TScoreSchemeAlign   scoringSchemeAlign;
+    TScoreScheme3       scoringSchemePreScoringFwd;
+    TScoreScheme3       scoringSchemePreScoringRev;
+    TScoreSchemeAlign   scoringSchemeAlignFwd;
+    TScoreSchemeAlign   scoringSchemeAlignRev;
```

Comment:
I kinda still prefer if we had `scoringSchemePreScoring` and `scoringSchemePreScoringBSRev` to clearly indicate that the second one is only used in the context of bisulphite stuff...